src/Mod/Sketcher/App/PythonConverter.cpp

```diff
@@ -59,11 +59,11 @@ std::string PythonConverter::convert(const Part::Geometry* geo, Mode mode)
     return command;
 }
 
-std::string PythonConverter::convert(const Sketcher::Constraint* constraint)
+std::string PythonConverter::convert(const Sketcher::Constraint* constraint, bool useLastGeoIdVar)

```

Comment:
I have a strong preference that we not add boolean parameters like this -- as @wwmayer pointed out to me a couple years ago, it results in a phenomenon called "boolean blindness" that makes code very difficult to understand later on. The way it is usually recommended is to use an enumeration that contains two clearly spelled-out options that a later reader will understand. That said, I'm not in a position to suggest good names here as I'm just learning about this part of the code.

Reply:
Ok I will find a proper name then.

Reply:
It is done